app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,30 @@
 class PostsController < ApplicationController
   def index
-    @users = User.find(params[:id])
-    @posts = @users.posts
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
   end
 
   def show
     @post = Post.find(params[:id])
   end
+
+  def new
+    @post = @current_user.posts.new
+  end
+
+  def create
+    @post = @current_user.posts.new(post_params)
+    if @post.save
+      @user = User.find(params[:user_id])
+      redirect_to user_posts_path(@user)
+    else
+      render :new
+    end
+  end
```

Comment:
- [ ] Kindly troubleshoot the post action and view to make it work, I tried to create a post but I failed, the app doesn't show any error. I have attached a screenshot for your reference.

![posts](https://github.com/Donmark2k/blog_app/assets/53124548/e7486a15-7060-4b5e-9f9c-0a4e5b839c4e)

Reply:
@elisha2kyakpo1 . if the first reviewer make changes, is the other changes not supposed to come as optional

Reply:
@Donmark2k, thank you for reaching out. The first reviewer did not test your app, the issue was about when the app loads, he was getting an error. It was difficult for him to create a post/comment or like.

Reply:
I have rectified the changes